cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/DefaultMetricStore.java

```diff
@@ -158,6 +158,12 @@ public Cube get() {
       ImmutableList.of(Constants.Metrics.Tag.NAMESPACE, Constants.Metrics.Tag.APP,
                        Constants.Metrics.Tag.SPARK)));
 
+    // batch adapters - can only contain mapreduce for now
+    aggs.add(new DefaultAggregation(
+      ImmutableList.of(Constants.Metrics.Tag.NAMESPACE, Constants.Metrics.Tag.ADAPTER),
```

Comment:
Shouldn't we aggregate by RunId (like the other aggregations)? Or is there no notion of a runId for adapters?
(A particular instance of an adapter type).